lib/Settings/Personal.php

```diff
@@ -33,12 +37,19 @@ public function getForm(): TemplateResponse {
 			$this->config->getAppValue(Application::APP_ID, 'text_to_image_picker_enabled', '1') === '1' :
 			false;
 		$textToImagePickerEnabled = $this->config->getUserValue($this->userId, Application::APP_ID, 'text_to_image_picker_enabled', '1') === '1';
+
+		$freePromptPickerAvailable = $freePromptTaskTypeAvailable ?
+			$this->config->getAppValue(Application::APP_ID, 'free_prompt_picker_enabled', '1') === '1' :
+			false;
+		$freePromptPickerEnabled = $this->config->getUserValue($this->userId, Application::APP_ID, 'free_prompt_picker_enabled', '1') === '1';
 		
 		$userConfig = [
-			'text_processing_available' => $textProcessingAvailable,
+			'assistant_available' => $assistantAvailable,
```

Comment:
Mistake or you don't want to name this `text_processing_available` anymore?

It does not really matter in the end as long as its meaning is explicit and clear.

Reply:
Here I thought that the difference is that in user settings it may be that text processing is available but that the admin has disabled the feature. That's why I thought it would make more sense here to name it assistant_available (since it may just be disabled by the admin).

Reply:
Hmm. Should it be top_bar_assistant_available/enabled here instead?

Reply:
Yes, whatever makes it very explicit on what it means and what it has an impact on